.github/workflows/release.yml

```
@@ -0,0 +1,165 @@
+name: release
+
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+env:
+  app-name: clj-xmas
+  repo: com.github.liquidz
+  pkg: merry
+
+jobs:
+  build:
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      matrix:
+        include:
+          - runs-on: ubuntu-latest
+            os: linux
+            cmd: gu
+            opts: "--static"
+          - runs-on: macOS-latest
+            os: darwin
+            cmd: gu
+            opts: ""
+          - runs-on: windows-latest
+            os: windows
+            cmd: gu.cmd
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install cl.exe
+        if: ${{ matrix.os == 'windows' }}
+        uses: ilammy/msvc-dev-cmd@v1
+      - uses: DeLaGuardo/setup-clojure@master
+        with:
+          cli: "1.10.3.1040"
+          lein: latest
+          # boot: latest
```

Leiningen は使っていないはずですが必要だったでしょうか？
また setup-clojure では `:cli` にも `latest` は指定可能なので `latest` の方が望ましいです
```suggestion
          cli: latest
```

---

cliがlatestの場合、Windowsにclojureがインストールされませんでした…

---

失敗(`cli: latest`): https://github.com/eggplants/clj-xmas/actions/runs/1597572281
成功(`latest`->`"1.10.3.1040"`): https://github.com/eggplants/clj-xmas/actions/runs/1597588106

---

https://github.com/DeLaGuardo/setup-clojure/issues/36

---

`https://download.clojure.org/install/win-install-latest.ps1`が存在せず、`https://download.clojure.org/install/win-install-1.10.3.1040.ps1`は存在しています。そこで落ちているようです。

---

https://github.com/DeLaGuardo/setup-clojure/blob/0fe110ad5aab7de04967d0f573267f7009aa0609/src/cli.ts#L79